package.json

```diff
@@ -32,5 +32,8 @@
     "text"
   ],
   "author": "olizilla",
-  "license": "BSD"
+  "license": "BSD",
+  "engines" : { 
+    "node" : ">=0.12" 
```

Comment:
@razbomi I was interested to find out when npm started supporting the `^` prefix and found this: https://gist.github.com/isaacs/64da1f4fcf8a1e21c347

If you're up for tweaking the `engines` definition to match the history i'll get this merged.